EMethod.h

```diff
@@ -0,0 +1,6 @@
+#ifndef __METHOD_HPP__
+#define __METHOD_HPP__
+
+enum EMethod { GET, POST, DELETE };
```

Comment:
지금 코드 작성하는 부분에도 수정했는데 EMethod를 EHttpMethod로 이름을 변경하는게 어떨까요?
```suggestion
enum EHttpMethod { GET, POST, DELETE };
```

Reply:
수정했습니다!